libs/core/src/lib/dialog/dialog-body/dialog-body.component.ts

```diff
@@ -19,11 +20,25 @@ import { DialogRef } from '../../dialog/utils/dialog-ref.class';
         '[class.fd-dialog__body--no-vertical-padding]': '!dialogConfig.verticalPadding'
     }
 })
-export class DialogBodyComponent {
+export class DialogBodyComponent implements AfterContentInit {
+
+    /** @hidden */
+    @ContentChild(WizardComponent)
```

Comment:
Consider adding `{ descendants: true }`

Reply:
I do not anticipate an instance of the wizard not being a direct descendant of the dialog body, but I will add this just in case